src/test/java/org/elasticsearch/indices/template/SimpleIndexTemplateTests.java

```diff
@@ -54,6 +56,7 @@
 /**
  *
  */
+@ClusterScope(scope = ElasticsearchIntegrationTest.Scope.TEST)
```

Comment:
I think this new test should be added to a separate new class, otherwise all of the existing methods will need to recreate the cluster before their execution, it's going to slow things down I'm afraid